adobe_vipm/flows/fulfillment.py

```diff
@@ -350,9 +351,11 @@ def _fulfill_termination_order(mpt_client, seller_country, order):
 
 def fulfill_order(client, order):
     logger.info(f'Start processing {order["type"]} order {order["id"]}')
-    seller_id = order["agreement"]["seller"]["id"]
-    seller = get_seller(client, seller_id)
-    seller_country = seller["address"]["country"]
+    if "parameters" in order:
+        order["parameters"] = unpack_structured_parameters(order["parameters"])
+    agreement = get_agreement(client, order["agreement"]["id"])
```

Comment:
Why you restored the get of the agreement ? In theory the poller send a select=+agreement and the API seems to understand it